packages/next/test/fixtures/00-app-dir-edge/vercel.json

```diff
@@ -17,7 +17,7 @@
       "headers": {
         "RSC": "1"
       },
-      "mustContain": "M1:{",
+      "mustContain": ":{",
```

Comment:
These changes were confirmed by @ijjk to be proper generalizations of the original assertion that still assert the original intent of the test.